voice/methods/about.md

```diff
@@ -12,13 +12,13 @@ Some of the URLs in this documentation contain placeholders for values that your
 
 ## REST API Reference Index
 
-| Resource                                                                                              | Description                                                        |
-|:------------------------------------------------------------------------------------------------------|:-------------------------------------------------------------------|
-| [`/calls`](calls/postCalls.md)                                                                        | The Calls resource lets you place phone calls.                     |
-| [`/calls/{callId}`](calls/postCallsCallId.md)                                                         | The CallsId resource lets you redirect active calls.               |
-| [`/calls/{callId}/recording`](calls/putCallsCallIdRecording.md)                                       | Pause or resume recording on an active call.                       |
-| [`/calls/{callId}/recordings`](calls/getCallsCallIdRecordings.md)                                     | Retrieve information about recordings that occurred during a call. |
-| [`/calls/{callId}/recordings/{recordingId}`](calls/getCallsCallIdRecordingsRecordingId.md)            | Retrieve information about a specific recording.                   |
-| [`/calls/{callId}/recordings/{recordingId}`](calls/deleteCallsCallIdRecordingsRecordingId.md)         | Delete a specific recording.                                       |
-| [`/calls/{callId}/recordings/{recordingId}/media`](calls/getCallsCallIdRecordingsRecordingIdMedia.md) | Retrieve a specific recording.                                     |
-| [`/recordings`](calls/getRecordings.md)                                                               | Retrieve information about your 1000 most recent recordings.       |
+| VERB                             | Resource                                                                                              | Description                                                                  |
```

Comment:
These are not verbs, these are methods

Reply:
Throughout the rest of our documentation (for both [messaging](https://github.com/Bandwidth/bandwidth.github.io/blob/stop-gap-v2/messaging/methods/about.md#media--attachment-management) and [applications](https://github.com/Bandwidth/bandwidth.github.io/blob/stop-gap-v2/account/applications/about.md#capabilities)), we refer to them as `Verbs`.
I agree that it should be changed, and I don't mind doing it, but I'm hesitant to include it in this story, since I think we'd need to change all of the occurrences in order to maintain consistency.